ocs/defaults.py

```diff
@@ -32,6 +32,8 @@
 CEPH_IMAGE = "ceph/ceph:v14.2.0-20190410"
 ROOK_IMAGE = "rook/ceph:master"
 DEPLOYMENT_PLATFORM = 'AWS'
+TEST_NAMESPACE = 'ocs-qe'
+GB = 1024 ** 3
```

Comment:
This should be in `constants.py` instead of `defaults.py`

Reply:
What does this default variable represent?